test/setup/pre_prepare.sh

```diff
@@ -12,8 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ruby '2.5.3'
-
-source 'https://rubygems.org/' do
-  gem 'kitchen-terraform', '~> 4.9'
-end
+# shellcheck disable=SC2018
+project_suffix="$(LC_ALL=C tr -dc 'a-z' < /dev/urandom | fold -w 4 | head -n 1)"
```

Comment:
It is better to generate this suffix inside terraform. Or use already created `local.suffix`